src/main/java/core/basesyntax/MainApp.java

```diff
@@ -1,5 +1,22 @@
 package core.basesyntax;
 
+import core.basesyntax.vehicles.Bulldozer;
+import core.basesyntax.vehicles.Excavator;
+import core.basesyntax.vehicles.Truck;
+
 public class MainApp {
 
+    public static void main(String[] args) {
+        Machine [] machines = {
+                new Truck(),
+                new Bulldozer(),
+                new Excavator()
+        };
```

Comment:
Use abstract references instead of specific ones. Instead of creating an array of `Machine` objects, you could declare them as `Machine truck = new Truck();` and so on, to utilize polymorphism effectively.